src/theme/index.tsx

```diff
@@ -17,6 +17,22 @@ export const MEDIA_WIDTHS = {
   deprecated_upToLarge: 1280,
 }
 
+const makeMediaQuery =
+  (size: number) =>
+  (...args: Parameters<typeof css>) =>
+    css`
+      @media (max-width: ${size}px) {
+        ${css(...args)}
+      }
+    `
+
+const deprecated_mediaWidthTemplates = {
```

Comment:
This should be explicitly typed still to stay in sync with `MEDIA_WIDTHS`: `: { [width in keyof typeof MEDIA_WIDTHS]: typeof css }`

Reply:
Good catch. We can use it to enforce any potential additions in the future.

Reply:
Decided to revert to original code with minimum changes, this is going to go away anyway in the nearest PRs